src/providers/ModalProvider.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { createContext, useState, FunctionComponent } from 'react';
+import { SocketContext } from './SocketProvider';
+
+export const ModalContext = createContext({
+  isOpen: false as boolean,
```

Comment:
```suggestion
  isOpen: boolean,
```
Rimelig sikker på at du ikke trenger `false` der i og med at du setter det i `useState<boolean>(false)`